src/core/libraries/np_trophy/np_trophy.cpp

```diff
@@ -249,19 +257,8 @@ int PS4_SYSV_ABI sceNpTrophyGetGameInfo(OrbisNpTrophyContext context, OrbisNpTro
 
     if (result) {
```

Comment:
This result branch can be made into early return
```
if (!result) {
    LOG_INFO(Lib_NpTrophy, "couldnt parse xml : {}", result.description());
    return ORBIS_OK;
}
```
However returning ORBIS_OK in this situation seems a bit risky, either appropriate error code or assert is probably better